recipes/tests/test_models.py

```diff
@@ -18,3 +26,54 @@ def test_str(self):
         expected = 'test_title'
         actual = Recipe.objects.get(title='test_title').__str__()
         self.assertEqual(expected, actual)
+
+    def test_upvote_count(self):
+        """
+        Validate total number of upvotes for a recipe
+        """
+        expected = 2
+        actual = self.recipe1.vote_objects.filter(up=True).count()
```

Comment:
this is a good test, but since we're actually going to be using the `upvotes` property on the object, it would be good to use that in the test as well. Make sure it has the expected number. You can have a test for each _way_ of getting the upvote count, or you can check that the filter count and the expected _and the upvotes_ property all have the same count